### PR TITLE
Scroll to top button

### LINKED ISSUE #107: Scroll to top button can be improved

### PR DESCRIPTION
**Pull request by shreyshreyansh**
Resolves #107 

**Before**

**After**

### CODE REVIEW

File: website/public/index.html [hunk 1]
```diff
@@ -15,6 +15,12 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
         <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+        <link
```
Comment (shreyshreyansh) [inline website/public/index.html:18]: cdn

File: website/src/components/Scroll/Scroll.css [hunk 2]
```diff
@@ -3,4 +3,5 @@
     bottom: 2vh;
     right: 5%;
     z-index: 2;
-}
\ No newline at end of file
+    border-radius: 100px;
```
Comment (shreyshreyansh) [inline website/src/components/Scroll/Scroll.css:6]: border-radius

File: website/src/components/Scroll/Scroll.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ const Scroll = ({ showBelow }) => {
           className="btn btn-primary toTop"
           onClick={scrollToTop}
         >
-          ^ Scroll to Top
+          <i class="fa fa-arrow-up" aria-hidden="true"></i>
```
Comment (shreyshreyansh) [inline website/src/components/Scroll/Scroll.js:33]: added arrow tag

**Review by nehakalbande — APPROVED** [review]
Looks more responsive now ✨⬆

**Pull request merged**